WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,15 +38,14 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v18.5-whats-new"
+msgctxt "v18.6-whats-new"
 msgid ""
-"This update makes it even easier to embed external content on your sites! Introducing the new dedicated embed blocks for Facebook, Instagram, Loom, and Smartframe. Plus, the editor automatically detects embeddable URLs you paste into an empty paragraph. Alongside these new features, we fixed an issue where some embedded URLs could not be edited and adjusted the preview cut-off so that you’ll always see the full embedded content.\n"
-"\n"
-"Speaking of blocks, you can now customize the text and background color in the Pullquote and Preformatted blocks.\n"
-"\n"
-"Need help reaching more readers? The new Publicize and Blogging Reminder nudges will help you stick to a consistent publishing schedule.\n"
-"\n"
-"Finally, we improved support for larger text sizes when choosing a homepage or page layout, fixed the site comments list not refreshing on certain occasions, and polished the pre-publishing bottom sheet header.\n"
+"- We've made it easier to see when there are new blocks available for use on your site\n"
+"- You can now highlight text using the <mark> HTML tag, as well as change the text and background color of the Search block button. Shiny!\n"
```

Comment:
@mokagio Thought: you might want to double-check at submission time that the use of unescaped `<mark>` didn't generate any bug at any point in our tooling pipeline. I actually have no idea if there is a risk that somewhere in the tooling this gets interpreted as a tag and maybe even removed, or if we already handle this properly without any issue?

I'm hoping this won't cause any issue in practice, but I figured it would be worth checking on ASC (after those will be translated and you'll run `deliver`) that the `<mark>` is still there (and without any odd escaping sequence erroneously introduced along the way)

Reply:
Good point. Will do that.